### DISCUSSION TITLE
Comparative benchmarking results

### DISCUSSION THREAD

**Discussion by dmbates** [JuliaStats/MixedModels.jl#712] [General]
I noticed that some of the benchmarking results show dramatic decreases in execution time and memory usage from the reference to the test runs.  I believe this is artificial and that the the reference and test runs are on the same version of Julia and MixedModels.  The difference is that the reference is run first and has the compilation overhead.

**Comment by dmbates** [comment]
I should have said `Target` and `Baseline` instead of `reference` and `test`.

**Comment by palday** [comment]
Yeah, I've noticed that the comparisons are extremely noisy as well. I stalled in #711 trying to find a set of benchmarks that are less noisy. It may be worthwhile to really reduce the portions that are actually being benchmarked, e.g. a call to `updateL!` instead of the entire fitting process.

**Comment by dmbates** [comment]
Agreed that benchmarking `updateL!` for a couple of large, complex examples will be more meaningful.
